src/Ssh.php

```diff
@@ -28,6 +30,8 @@ public function __construct(string $user, string $host, int $port = null)
             $this->usePort($port);
         }
 
+        $this->removeBash = false;
```

Comment:
Let's make this implementation positive by default. Replace this line with

```
$this->addBash = true
```

Reply:
Updated it to `addBash`